packages/pyolite-kernel/src/worker.ts

```diff
@@ -109,8 +109,8 @@ async function getpass(prompt: string) {
   const replyPromise = new Promise(resolve => {
     resolveInputReply = resolve;
   });
-  const result = await replyPromise;
-  return result;
+  const result: any = await replyPromise;
+  return result['value'];
```

Comment:
You will need to call `toPy` to make it a Python object

Reply:
We can directly access `value` which is a string and thus don't need a conversion.